### PR TITLE
Ajax Calls

### PR DESCRIPTION
**Pull request by divpaul2016**

### CODE REVIEW

File: js/stgeorge.js [hunk 1]
```diff
@@ -51,19 +51,204 @@ $("input[name='reset_button']").click(function() {
...
+            "phonenumber": form.find('input[name="phone"]'),
+            "emailaddress": form.find('input[name="email"]'),
+            "message": form.find('input[name="prayer"]',
+			"services": form.find('input[name="selection"]',
+            "requestfor": "qurbananominations"  
+           }),
+
+      success: function(msg) {
+        enableButtons();
+        $("#sendmessage").addClass("show");
```
Comment (josephjoys) [inline js/stgeorge.js:72]: Do we have #sendmessage element in the UI?

File: js/stgeorge.js [hunk 2]
```diff
@@ -51,19 +51,204 @@ $("input[name='reset_button']").click(function() {
...
+        function clear() {
+             $("#sendmessage").removeClass("show");
+        }
+        setTimeout(clear, 2000);
+
+        document.getElementById('first_name').value="";
```
Comment (josephjoys) [inline js/stgeorge.js:119]: This will not work. Our form elements use Name not Id

File: js/stgeorge.js (hunk 1 above)
Comment (divpaul2016) [reply js/stgeorge.js:72]: i just copied the code from Contact.js .. I do not know what to do .. Please help me here

File: js/stgeorge.js (hunk 2 above)
Comment (divpaul2016) [reply js/stgeorge.js:119]: fixed

File: js/stgeorge.js (hunk 1 above)
Comment (josephjoys) [reply js/stgeorge.js:72]: I think this will not work for contact also.

**Pull request merged**